fe/fe-core/src/main/java/com/starrocks/catalog/ScalarType.java

```diff
@@ -50,9 +50,9 @@ public class ScalarType extends Type implements Cloneable {
     // precision that is supported by the smallest decimal type in the BE (4 bytes).
     public static final int DEFAULT_PRECISION = 9;
     public static final int DEFAULT_SCALE = 0; // SQL standard
-    // Longest supported VARCHAR and CHAR, chosen to match Hive.
-    public static final int DEFAULT_STRING_LENGTH = 65533;
     public static final int MAX_VARCHAR_LENGTH = 1048576;
+    // Longest supported VARCHAR and CHAR, chosen to match Hive.
+    public static final int DEFAULT_STRING_LENGTH = MAX_VARCHAR_LENGTH;
```

Comment:
I think you can't change DEFAULT_STRING_LENGTH, because for native engine, we can't support too long string. so we keep this default string length to be 65533. If you want to use longer string, you can use a new variable not this.